nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-administration/src/main/java/org/apache/nifi/admin/dao/impl/StandardKeyDAO.java

```diff
@@ -156,13 +156,13 @@ public Key createKey(final String identity) {
     }
 
     @Override
-    public void deleteKeys(String identity) {
+    public Integer deleteKeys(String identity) {
         PreparedStatement statement = null;
         try {
             // add each authority for the specified user
             statement = connection.prepareStatement(DELETE_KEYS);
```

Comment:
If we change the statement to use the `ID` rather than `IDENTITY`, this should be consistent across all identity providers, correct?

`DELETE FROM KEY WHERE ID = 1` vs. `DELETE FROM KEY WHERE IDENTITY = 'admin'` (or possibly `IDENTITY = 'admin@REALM'`).